api/models.py

```diff
@@ -2,18 +2,20 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.contrib.postgres.fields import JSONField
+from typedmodels.models import TypedModel
 
 
 class User(AbstractUser):
     id = models.AutoField(primary_key=True)
     gravatar = models.URLField(blank=True)
 
 
-class Collection(models.Model):
+class CollectionBase(TypedModel):
     title = models.TextField()
     description = models.TextField(null=True, blank=True)
     tags = models.TextField(null=True, blank=True)
     created_by = models.ForeignKey(User)
+    created_by_org = models.TextField(null=True, blank=True)
```

Comment:
That's the assumption I've always been under, because it takes up a predefined, smaller amount of space in the database (especially if the max length is 255 chars or shorter), so title would be a CharField and description would be a TextField. OSF uses TextField for node titles, but it seems like the reason for this is to handle pre-Django node titles that might be ridiculously long, and that the title, without that possible migration issue, have been a CharField.